From c6121eb3ca44f5a448b7a2d5683ac52caa30cbe0 Mon Sep 17 00:00:00 2001
From: Charles Mirabile <cmirabil@redhat.com>
Date: Wed, 9 Apr 2025 13:08:03 -0400
Subject: [PATCH 40/74] fixup! drivers: reset: Add reset driver for EIC770X SoC

Added missing module author / license / description so that
this driver can be built as a module. I added them based on
the comments from the top of the file.
---
 drivers/reset/reset-eswin.c | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/drivers/reset/reset-eswin.c b/drivers/reset/reset-eswin.c
index 0dd44c228b53..3fda8842fc3b 100755
--- a/drivers/reset/reset-eswin.c
+++ b/drivers/reset/reset-eswin.c
@@ -253,3 +253,7 @@ static int __init eic7700_reset_init(void)
 	return platform_driver_register(&eswin_reset_driver);
 }
 arch_initcall(eic7700_reset_init);
+
+MODULE_AUTHOR("Huangyifeng <huangyifeng@eswincomputing.com>");
+MODULE_DESCRIPTION("Eswin Reset Controller Driver for EIC770X SoCs");
+MODULE_LICENSE("GPL v2");
-- 
2.52.0

